common/remote/remoteerr.hpp

```diff
@@ -86,6 +87,7 @@
 #define RAERR_InvalidUsernamePassword_Text      "Invalid (upper case U) in username/password"
```

Comment:
I dunno, no change here, apart from spurious new blank line below it (which I'll remove)